src/makeRobot.js

```diff
@@ -38,7 +38,70 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    'name': name,
+    'wheels': wheels,
+    'chip version': version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(num = 1) {
+      if (num < 0) {
+        return robot;
+      } else {
+        this.coords.y += num;
+
+        return robot;
+      }
```

Comment:
Let's make logic here and in other methods more simple.
```suggestion
      if (num >= 0) {
        this.coords.y += num;
      }
      
      return robot;
```